docs/documentation/server_admin/topics/users/con-aia.adoc

```diff
@@ -39,18 +39,35 @@ by checking the claims like `acr` in the tokens.
 [id="con-aia-reauth_{context}"]
 == Re-authentication during AIA
 
-In case the user is already authenticated due to an active SSO session, that user usually does not need to actively re-authenticate. However, if that user actively authenticated longer than five minutes ago,  
+In case the user is already authenticated due to an active SSO session, that user usually does not need to actively re-authenticate. However, if that user actively authenticated longer than five minutes ago,
 the client can  still request re-authentication when some AIA is requested. Exceptions exist from this guideline as follows:
 
 * The action `delete_account` will always require the user to actively re-authenticate
 
 * The action `update_password` might require the user to actively re-authenticate according to the configured <<maximum-authentication-age,Maximum Authentication Age Password policy>>.
-In case the policy is not configured, it also defaults to five minutes.
+In case the policy is not configured, it is also possible to configure it on the required action itself in the <<proc-setting-default-required-actions_{context}, Required actions tab>>
+when configuring the particular required action. If not configured anywhere from those places, it defaults to five minutes.
 
 * If you want to use a shorter re-authentication, you can still use a parameter query parameter such as  `max_age` with the specified shorter value or eventually `prompt=login`, which will always require user to
 actively re-authenticate as described in the OIDC specification. Note that using `max_age` for a longer value than the default five minutes (or the one prescribed by password policy) is not supported.
 The `max_age` can be currently used only to make the value shorter than the default five minutes.
 
+* In case that <<_step-up-flow,Step-up authentication>> is enabled and the action is either adding the new credential or deleting the existing credential, then it is required to authenticate with the level corresponding
+to the given credential in case that user already has the credential of the particular level. For example if there are `otp` and `webauthn` configured in authentication flow as 2nd-factor authenticators
```

Comment:
```suggestion
* If <<_step-up-flow,Step-up authentication>> is enabled and the action is to add or delete a credential, authentication is required with the level corresponding
to the given credential. This requirement exists in case the user already has the credential of the particular level. For example, if `otp` and `webauthn` are configured in authentication flow as 2nd-factor authenticators
```